pytorch_lightning/callbacks/base.py

```diff
@@ -273,11 +273,21 @@ def on_save_checkpoint(
         """
         pass
 
-    def on_load_checkpoint(self, callback_state: Dict[str, Any]) -> None:
+    def on_load_checkpoint(
+        self, trainer: 'pl.Trainer', pl_module: 'pl.LightningModule', callback_state: Dict[str, Any]
+    ) -> None:
         """Called when loading a model checkpoint, use to reload state.
 
         Args:
+            trainer: the current Trainer instance.
+            pl_module: the current 'pl.LightningModule' instance.
             callback_state: the callback state returned by ``on_save_checkpoint``.
+
+        .. note::
+
```

Comment:
```suggestion
        Note:

```
Lightning uses Google style docstrings 
https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html